src/subscriptions/SubscriptionManager.h

```diff
@@ -41,7 +41,7 @@ class Subscription
     Subscription(Subscription&) = delete;
     Subscription(Subscription&&) = delete;
 
-    explicit Subscription(boost::asio::io_context& ioc) : strand_(ioc.get_executor())
+    explicit Subscription(boost::asio::io_context& ioc) : strand_(boost::asio::make_strand(ioc))
```

Comment:
we need to call extra copy constructor after change?

Reply:
thanks to guaranteed copy-elision this is ok. it's just a nicer way to say the same thing without explicitly using get_executor.